tools/sgcollect_info

```diff
@@ -263,7 +265,7 @@ def make_collect_logs_tasks(zip_dir, sg_url, salt, should_redact):
 
     # Find log file path from logging.["default"] style config
     guessed_logging_path = extract_element_from_default_logging_config('LogFilePath', config_str)
-    if guessed_logging_path:
+    if guessed_logging_path and os.path.isfile(guessed_logging_path):
```

Comment:
As discussed, we still need to handle both cases - when LogFilePath is a path or a filename.  For a filename, we can use a similar approach to the one above.  For a path, we can use a similar approach to the 2.1 style to pick up rotated logs.

Reply:
Make sense, refactored the make_collect_logs_tasks function to handle both cases.